string-manipulation-2/src/num-vowels.js

```diff
@@ -0,0 +1,12 @@
+/* exported numVowels */
+function numVowels(string) {
+  var number = 0;
+  var vowels = ['a', 'e', 'i', 'o', 'u', 'A', 'E', 'I', 'O', 'U'];
+  for (var i = 0; i < string.length; i++) {
+    for (var j = 0; j < vowels.length; j++) {
+      if (string[i] === vowels[j]) {
+        number++;
+      }
+    }
+  } return number;
```

Comment:
You can also accomplish this using a `switch` statement, that way you don't need to use a nested for loop
```
function numVowels(string) {
  var vowelCount = 0;
  for (var i = 0; i < string.length; i++) {
    switch (string[i].toLowerCase()) {
      case 'a':
      case 'e':
      case 'i':
      case 'o':
      case 'u':
        vowelCount++;
    }
  }
  return vowelCount;
}
```